.github/workflows/pr-auto-label.yml

```
@@ -0,0 +1,82 @@
+                console.log('Error removing labels:', error.message);
+              }
+
+              // 然后添加新的标签
+              await github.rest.issues.addLabels({
+                issue_number: context.issue.number,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                labels: labelsArray
+              });
+              console.log('New labels added successfully');
+            }
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)